cpp/modmesh/toggle/toggle.hpp

```diff
@@ -43,6 +43,14 @@ class Toggle
 
 public:
 
+    static constexpr bool USE_PYSIDE =
```

Comment:
I am not giving `USE_PYSIDE` an address so that it will not be possible to accidentally use it during runtime.  It should always be replaced during compile time.